README.md

```diff
@@ -43,7 +43,7 @@ When you want to use executeCommand in a subdirectory, make sure to cd there fir
 
 ## API Endpoints
 
-View API endpoints that allow you to interact with the file system within the Kaguya directory [here](pages/apiEndpoints.md).  
+View [API endpoints](https://github.com/ThirDecade2020/kaguya/blob/master/pages/apiEndpoints.md) that allow you to interact with the [FILES directory](https://github.com/ThirDecade2020/kaguya/tree/master/FILES) by default.
```

Comment:
This link should be formatted so it works when it's merged

Reply:
OK had a thought and I see what's happening here some q's if you don't mind?

1) All my links throughout the pull request are wrong because they are either incomplete or link user to my forked version of the repository.

2) 1 link should take user to a newly created 'apiEndpoints.md', and 1 link should take user to 'FILES' which already existed pre-PR. 'FILES' link is easy to find in original Kaguya, but newly created 'apiEndpoints.md' won't appear in original Kaguya until after merge. My question is, should we insert a link into a PR that goes to a page that won't be created until after the merge? Should I do separate pull requests, A. create new page first, and then B. create link to that new page?

3) Lastly, I didn't realize this, but at the beginning of this PR I did not add a new branch to my version of the repository so I am working directly from my Main branch. Should I start this PR over and create a new branch for each individual change? 

Your help and patience is much appreciated and I'll continue working on this.